src/modules/ui.js

```diff
@@ -0,0 +1,95 @@
+// UI Class: Handle UI Tasks
+import Task from './task.js';
+import Store from './store.js';
+
+class UI {
+  static displaytasks = () => {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => UI.addTaskToList(task));
+  }
+
+  static getNewTask = () => {
+    const newTask = document.querySelector('#new-task').value;
+
+    // valitdate empty form
+    if (newTask !== '') {
+      // making object of Book class
+      const task = new Task(newTask);
+
+      // add book to UI
+      UI.addTaskToList(task);
+
+      // add book to store
+      Store.addTask(task);
+
+      // clear fields
+      UI.clearFields();
+    }
+  }
```

Comment:
while comments can be helpful to document the purpose and functionality of our  code 👍  ,  too much use comments don't make our codebase look good so I kindly request you remove some of the comments in your code